### ISSUE TITLE
Guide to installation

### ISSUE THREAD

**Issue by Predad0rAlfa** [singh-vikas-m/live-record-replay-website#1]
Thank you very much for the contribution, you could give write a detailed guide on how to install and configure on the server

**Comment by VikieVik** [comment]
Hey just go to server folder, and do npm install once done do npm start

**Comment by Predad0rAlfa** [comment]
i understood, i ran the npm install command and the packages were installed, but how do I access the environment to view the recordings and get the tag to insert on the site and receive the recordings

**Comment by Predad0rAlfa** [comment]
@singh-vikas-m @@

print 

https://ibb.co/0Cptzcp

**Comment by VikieVik** [comment]
I have added loom video in readme pls refer that

**Comment by VikieVik** [comment]
@Predad0rAlfa !👆

**Comment by Predad0rAlfa** [comment]
HI @singh-vikas-m thank you very much for the time you dedicated to the video, but I thought that the project would record each sessao, can you add this option to the code?

option to record each sessao and inspect each sessao would be divine !!

If you need any incentive I can contribute financially

**Comment by VikieVik** [comment]
This repo was dedicated for live session but I can do some adjustment and add both recoding mode as well. Eg rather than just streaming events I'll also store them in database. That can be replayed. Give me couple of days.

**Comment by frankcom** [comment]
Could it be that the server started on localhost is blocked for accessing on some systems? I think I followed all the steps in video, but I can not see the live recording. Thanks for doing such great work.

**Comment by VikieVik** [comment]
Try checking network logs

**Comment by frankcom** [comment]
Thanks!

npm WARN crony-socket-io-server@1.0.0 No description
npm WARN crony-socket-io-server@1.0.0 No repository field.

just saw this when doing install again.

Could this cause the problem?

This comes when starting:

> crony-socket-io-server@1.0.0 start /home/achats/public_html/recor/1/server
> node app.js

socket io server started......
(node:3010) Warning: N-API is an experimental feature and could change at any time.

**Comment by VikieVik** [comment]
Nope. Just a warning. What's the exact issue mind sharing a loom video

**Comment by frankcom** [comment]
Hi thanks.
https://www.loom.com/share/18abcf32f23f4e5bba3de4f0e987f80e
Hope it works!

**Comment by VikieVik** [comment]
Looks good. Make sure the player website is loaded first then the client
website. Eg: when both the sites are loaded completely, refresh the
replayer site once done, refresh the client site and you'll see the live
replay

**Comment by VikieVik** [comment]
I'm assuming that you have already started socket server code before
starting both websites

**Comment by frankcom** [comment]
Yes I have, but not working yet. 
I have just sent an email.
